src/foundation/y_check_base.clas.abap

```diff
@@ -711,11 +711,9 @@ CLASS Y_CHECK_BASE IMPLEMENTATION.
 
 
   METHOD get_class_description.
-    TRY.
-        result = NEW cl_oo_class( myname )->class-descript.
-      CATCH cx_class_not_existent.
-        result = 'Description Not Available'.
-    ENDTRY.
+    SELECT SINGLE descript INTO @DATA(description) FROM seoclasstx WHERE clsname = @myname.
+    result = COND #( WHEN description IS NOT INITIAL THEN description
+                     ELSE 'Description Not Available' ).
```

Comment:
you can write it that way:

```abap
SELECT SINGLE descript INTO @result FROM seoclasstx WHERE clsname = @myname.
IF sy-subrc <> 0.
result = 'Description Not Available'.
ENDIF.
```
If the description is avialble then it writes the entry into result.
if not, you get an subrc <> 0 and return the text.

Reply:
Done :)